Frontend/CO.CDP.OrganisationApp/Pages/OrganisationOverview.cshtml

```diff
@@ -6,6 +6,9 @@
 @{
     var organisationDetails = Model.OrganisationDetails!;
     var registeredAddress = organisationDetails.Addresses.FirstOrDefault(a => a.Type == Organisation.WebApiClient.AddressType.Registered);
+    var pponIndentifier = organisationDetails.AdditionalIdentifiers.Prepend(organisationDetails.Identifier).FirstOrDefault(i => i.Scheme == "CDP-PPON");
+    var otherIdentifier = organisationDetails.AdditionalIdentifiers.Prepend(organisationDetails.Identifier).FirstOrDefault(i => i.Scheme == "Other");
```

Comment:
This isn't quite right.

We should only show PPON if it's the only identifier we have (ignoring "Other").

Otherwise, we should show the primary identifier here.

Reply:
Oh, I guess that's what happens! I misunderstood "otherIdentifier"'s purpose.

Reply:
It's ok, I thought Sammy wanted us to show a PPON if there was an Other?